webdrivercss-pull64.patch

```diff
@@ -0,0 +1,5 @@
+21c21,22
+<             excludeRect: []
+---
+>             excludeRect: [],
+>             scrollPos: {x: 0, y:0}, 
```

Comment:
Until the webdriverio folks merge https://github.com/webdriverio/webdrivercss/pull/64/files we can patch it manually with `cat webdrivercss-pull64.patch | patch -p0 /path/to/getPageInfo.js`